src/libdredd/src/mutate_visitor.cc

```diff
@@ -258,6 +258,30 @@ bool MutateVisitor::HandleUnaryOperator(clang::UnaryOperator* unary_operator) {
     return true;
   }
 
+  // There is no useful way to mutate this expression.
+  if (optimise_mutations_ && ((unary_operator->getOpcode() == clang::UO_Minus &&
```

Comment:
This boolean expression is too big for my taste :)

Can you please refactor to:

```
if (optimise_mutations_) {
   if (opcode is minus && (disjunction of conditions)) {

   }
   if (opcode is not && (disjunction of conditions)) {

   }
}
```

Reply:
Done.